recipes-support/snapd/snapd_2.56.2.bb

```diff
@@ -1,150 +1,4 @@
-SUMMARY = "The snapd and snap tools enable systems to work with .snap files."
-HOMEPAGE = "https://www.snapcraft.io"
-LICENSE = "GPL-3.0-only"
-LIC_FILES_CHKSUM = "file://${WORKDIR}/${PN}-${PV}/COPYING;md5=d32239bcb673463ab874e80d47fae504"
-
-SRC_URI = "	\
-	https://${GO_IMPORT}/releases/download/${PV}/snapd_${PV}.vendor.tar.xz	\
-  file://0001-mkversion-data-generate-supported-assert-formats-inf.patch \
-"
+include snapd.inc
```

Comment:
thanks for doing this. I had it in my TODO list

Reply:
I have a couple more changes coming shortly.